tests/puppeteer/campaigns/functional/BO/design/pages/03_categoriesPagesBulkAction.js

```diff
@@ -0,0 +1,289 @@
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const PagesPage = require('@pages/BO/design/pages/pages');
+const AddPageCategoryPage = require('@pages/BO/design/pages/addPageCategory');
+const AddPagePage = require('@pages/BO/design/pages/addPage');
+const CategoryFaker = require('@data/faker/CMScategory');
+const PageFaker = require('@data/faker/CMSpage');
+
+let browser;
+let page;
+let numberOfCategories = 0;
+let numberOfPages = 0;
+let firstCategoryData;
+let secondCategoryData;
+let firstPageData;
+let secondPageData;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    pagesPage: new PagesPage(page),
+    addPageCategoryPage: new AddPageCategoryPage(page),
+    addPagePage: new AddPagePage(page),
+  };
+};
+
+// Create Categories, Then disable / Enable and Delete with Bulk actions
+describe('Create Categories/ Pages, Then disable / Enable and Delete with Bulk actions', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+    firstCategoryData = await (new CategoryFaker({name: 'todelete'}));
+    secondCategoryData = await (new CategoryFaker({name: 'todelete'}));
+    firstPageData = await (new PageFaker({title: 'todelete'}));
+    secondPageData = await (new PageFaker({title: 'todelete'}));
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to Pages page
+  loginCommon.loginBO();
+  it('should go to "Design>Pages" page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.designParentLink,
+      this.pageObjects.boBasePage.pagesLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.pagesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.pagesPage.pageTitle);
+  });
+  it('should reset filter and get number of categories in BO', async function () {
+    if (await this.pageObjects.pagesPage.elementVisible(
+      this.pageObjects.pagesPage.categoryfilterResetButton, 2000)) {
+      await this.pageObjects.pagesPage.resetFilter(this.pageObjects.pagesPage.categoryfilterResetButton);
+    }
+    numberOfCategories = await this.pageObjects.pagesPage.getNumberFromText(
+      this.pageObjects.pagesPage.categoryGridTitle);
+    if (numberOfCategories === 0) {
+      const filterButtonVisibility = await this.pageObjects.pagesPage.elementVisible(
+        this.pageObjects.pagesPage.categoryfilterResetButton, 2000);
```

Comment:
```suggestion
        this.pageObjects.pagesPage.categoryfilterResetButton, 
        2000
      );
```

Reply:
```suggestion
        this.pageObjects.pagesPage.categoryfilterResetButton,
        2000,
     );
```

Reply:
Thanks I did it directly in github :p (don't have eslint integrated yet :( )